docs/Coding Environment/deployment-to-production.md

```diff
@@ -81,7 +81,8 @@ You need to set these values before deploying sandbox or production. We store th
 
 To set up deploy values:
 1. Make sure you are on the desired branch
-2. Run `forge update-values`
-3. Update values, save and exit
-4. Press enter in the terminal to upload values to our CI server
-5. You can now deploy the branch
+2. Run `forge update-deployment-values`
+3. Update values with `export key='value'`. (Every export statement has to be in a new line)
+4. Save (Ctrl+o) and Exit (Ctrl+x)
+5. Press enter in the terminal to upload values to our CI server
```

Comment:
In connecting-git "enter" is written in caps

Reply:
Updated